ipi_tests/regression_tests/tests/NPT/gas-npzt-VT/input.xml

```diff
@@ -0,0 +1,60 @@
+<simulation threading='false'>
+    <ffsocket mode='unix' name='driver'>
```

Comment:
maybe name = "gas" and address: localhost to follow the conventions of the previous tests?